test/test_main.c

```diff
@@ -0,0 +1,24 @@
+#include "test.h"
+#include "proc/threads.h"
+
+DECLARE_TEST_GROUP(test_list);
```

Comment:
we should probably comment that it's necessary to explicitly declare new test groups here